### ISSUE TITLE
Submit Button does not work, Unknown User at the bottom of bot page, Multiple Owners does not work too

### ISSUE THREAD

**Issue by shootdot** [Sank6/Discord-Bot-List#153]

**Comment by Sank6** [comment]
The submit button for what?

**Comment by shootdot** [comment]
> The submit button for what?

Submit bot

**Comment by Sank6** [comment]
The submit button works for me. Are you sure you have valid config setup? The multiple owners thing is an issue though.

**Comment by shootdot** [comment]
What config, my config.json is ok but When I click submit it doesn't show success or error screen but bot gets added to queue.

**Comment by shootdot** [comment]
And toggle them also don't work :(

**Comment by shootdot** [comment]
Unknown user 
[IMAGE]https://user-images.githubusercontent.com/55920587/91252499-c40fa580-e76e-11ea-8d76-f0fac55d1d1b.png

**Comment by Sank6** [comment]
I don't know what you're doing differently, but toggle theme works fine for me. Owners also shows up. How are you hosting this?

**Comment by shootdot** [comment]
Ok now it's working now but when we verify it verifies by shows an error
memv not define and queue is bugged and once the data dissent get replaced
it uses the old data Example if I submit 1 bot and I remove it then other
person submits the bot it shows the old ow ers name that's me.

**Comment by Sank6** [comment]
It seems like your database might be corrupted. How are you hosting this again?

**Comment by shootdot** [comment]
*I am hosting in GoDaddy domain with AWS UBUNTU 18 Instance using local
MongoDB and doing DNS forwarding *

**Comment by shootdot** [comment]
@Sank6 Is embed still under development ?
![739895373945897000](https://user-images.githubusercontent.com/55920587/91651040-75c51400-ea98-11ea-9d3b-9f129576c1d4.png)
I get like this

**Comment by Sank6** [comment]
No, embed works fine too. I have no idea how you're getting all this. What's your node version?

**Comment by shootdot** [comment]
Node v12.18.3

**Comment by Sank6** [comment]
Try cloning the repo again, keeping the config file.

**Comment by shootdot** [comment]
It's fixed the font config file was missing thanks for your help :)

**Comment by shootdot** [comment]
```
            "bot": "",
            "unverified": "",
            "verified": "",
            "unmuted": "",

            "bot_developer": "",
            "bot_verifier": ""
```
what does this values mean ?

**Comment by Sank6** [comment]
Those are role IDs. Make roles in your server and paste in those IDs

**Comment by shootdot** [comment]
> Those are role IDs. Make roles in your server and paste in those IDs

I know that are role ids, but verified for what unverified for what like that I am asking please can u comment an properly mention what are those role ids for eg for bot or member

**Comment by Sank6** [comment]
Bot - General role for all bots
Verified - Role that bots get once they're verified by you or other bot verifiers
 Unverified - Role that bots get as soon as they join the server to be tested
Unmuted - All bots that aren't muted get this role. (may change soon and use a muted role)

**Comment by shootdot** [comment]
Login Button error
Once I login still it shows the login button but still I can access my submitted bots

**Comment by Sank6** [comment]
Yes that has been posted in another issue. I'm working on a fix.
